hierarchy/entity.h

```diff
@@ -0,0 +1,23 @@
+//CS300 group #2
+//
+//Worked on by Tristan Gomez (Started 2/5/2020)
+
+#include <iostream>
+#include "tString.h"
+const int ID = 9;
+
+class entity
+{
+	public:
+		entity();
+		entity(const entity & toAdd);
+		~entity();
+		bool addId();
```

Comment:
Using 'add' for setters and simultaneously using it for user input is unintuitive.